client-js/README.md

```diff
@@ -0,0 +1,202 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+## Installing Dependencies
+
+Dependencies can be installed using the following command in the `client-js` directory:
+
+```js
+npm i
+```
+
+## Installing a CEP-78 Contract using the JavaScript Client
+
+The `install` method crafts a [Deploy](https://docs.casperlabs.io/design/casper-design/#execution-semantics-deploys) using `InstallArgs` and sends that Deploy to the specified Casper network.
+
+```js
+
+const install = async () => {
+  const cc = new CEP78Client(process.env.NODE_URL!, process.env.NETWORK_NAME!);
+
+  const installDeploy = await cc.install(
+    {
+      collectionName: "my-collection",
+      collectionSymbol: "MY-NFTS",
+      totalTokenSupply: "1000",
+      ownershipMode: NFTOwnershipMode.Transferable,
+      nftKind: NFTKind.Physical,
+      jsonSchema: {
+        properties: {
+          color: { name: "color", description: "", required: true },
+          size: { name: "size", description: "", required: true },
+          material: { name: "material", description: "", required: true },
+          condition: { name: "condition", description: "", required: false },
+        },
+      },
+      nftMetadataKind: NFTMetadataKind.CustomValidated,
+      identifierMode: NFTIdentifierMode.Ordinal,
+      metadataMutability: MetadataMutability.Immutable,
+      mintingMode: MintingMode.Installer,
+    },
+    "165000000000",
+    FAUCET_KEYS.publicKey,
+    [FAUCET_KEYS]
+  );
+
+  const hash = await installDeploy.send(process.env.NODE_URL!);
+
+```
+
+`InstallArgs` are specified in the associated `.env` file as follows:
```

Comment:
the `.env` is used only for testing purposes so it's not connected with the whole client.